src/index.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import configureStore from './redux/configureStore';
 import App from './App';
 
 ReactDOM.render(
```

Comment:
- [ ] OPTIONAL
Your app is not rendering as intended.

**Render error:**
![image](https://user-images.githubusercontent.com/12745474/163263060-79abaf22-1b3e-4f65-a7a3-260e8c56cc23.png)


To improve upon the above kindly update your _index.js_ file to look with React 18 syntax below:

```ruby
import { StrictMode } from 'react';
import { Provider } from 'react-redux';
import store from './redux/configureStore';
import App from './components/App';
import { createRoot } from "react-dom/client";

const rootElement = document.getElementById("root");
const root = createRoot(rootElement);

root.render(
  <Provider store={store}>
    <StrictMode>
      <App />
    </StrictMode>
  </Provider>,
);

```